ui/components/app/account-menu/index.scss

```
@@ -10,7 +10,7 @@
   color: var(--white);
 
   @media screen and (max-width: $break-small) {
-    right: calc(((100vw - 100%) / 2) + 8px);
+    right: 1rem;
```

I think we should move towards a convention of using `px` values for bounding box attributes e.g. `padding` and `margin`. `rem`s/`em`s don't provide any accessibility benefit for things other than fonts. Pixels are also more aligned with how designers work. Have some notes on CSS guidelines but have yet to publish them anywhere will add them to the current documentation effort. Also open to discussions on this!

```suggestion
    right: 16px;
```